modular_skyrat/modules/company_imports/code/armament_datums/sol_defense.dm

```diff
@@ -2,21 +2,59 @@
 	category = SOL_DEFENSE_DEFENSE_NAME
 	company_bitflag = CARGO_COMPANY_SOL_DEFENSE
 
-// Basic armor vests
+// Armor vests for protecting against bullets
 
 /datum/armament_entry/company_import/sol_defense/armor
-	subcategory = "Light Body Armor"
+	subcategory = "Ballistic Armor"
+	cost = PAYCHECK_CREW * 3
+
+/datum/armament_entry/company_import/sol_defense/armor/ballistic_helmet
+	item_type = /obj/item/clothing/head/helmet/sf_peacekeeper/debranded
+
+/datum/armament_entry/company_import/sol_defense/armor/sf_ballistic_helmet
+	item_type = /obj/item/clothing/head/helmet/sf_peacekeeper
+
+/datum/armament_entry/company_import/sol_defense/armor/soft_vest
+	item_type = /obj/item/clothing/suit/armor/sf_peacekeeper/debranded
+
+/datum/armament_entry/company_import/sol_defense/armor/sf_soft_vest
+	item_type = /obj/item/clothing/suit/armor/sf_peacekeeper
+
+/datum/armament_entry/company_import/sol_defense/armor/normal_vest
+	name = "type I vest - normal"
+	item_type = /obj/item/clothing/suit/armor/vest/alt
 
 /datum/armament_entry/company_import/sol_defense/armor/slim_vest
 	name = "type I vest - slim"
 	item_type = /obj/item/clothing/suit/armor/vest
+
+/datum/armament_entry/company_import/sol_defense/armor_hardened
+	subcategory = "Hardened Armor"
 	cost = PAYCHECK_CREW * 3
 
-/datum/armament_entry/company_import/sol_defense/armor/normal_vest
-	name = "type I vest - normal"
-	item_type = /obj/item/clothing/suit/armor/vest/alt
+/datum/armament_entry/company_import/sol_defense/armor_hardened/enclosed_helmet
+	item_type = /obj/item/clothing/head/helmet/toggleable/sf_hardened
+
+
+/datum/armament_entry/company_import/sol_defense/armor_hardened/emt_enclosed_helmet
+	item_type = /obj/item/clothing/head/helmet/toggleable/sf_hardened/emt
+
+/datum/armament_entry/company_import/sol_defense/armor_hardened/hardened_vest
+	item_type = /obj/item/clothing/suit/armor/sf_hardened
+
+/datum/armament_entry/company_import/sol_defense/armor_hardened/emt_hardened_vest
+	item_type = /obj/item/clothing/suit/armor/sf_hardened/emt
```

Comment:
Why would I take the soft vests if the hardened vests cost the exact same amount of credits?

Reply:
The "soft" vests have significantly higher wound protection (twice the amount!)